source/cps/install.rst

```diff
@@ -20,8 +20,24 @@
         $ mkdir -p ~/src/
         $ mv PROGRAMS.330 ~/src/CPS
 
+3.  安装依赖
 
-3.  配置:
+    编译 CPS 源码需要安装 X11 库文件::
```

Comment:
Windows?

Reply:
Do you mean WSL users?

Reply:
yes.

Reply:
It's just the same as normal Linux distros.

Reply:
really? I remember one student cannot run `taup` when they use wsl without x server.

Reply:
See https://github.com/seismo-learn/seismology101/issues/90#issuecomment-765020467